libs/openFrameworks/gl/ofFbo.cpp

```diff
@@ -934,16 +934,21 @@ void ofFbo::updateTexture(int attachmentPoint) {
 			glPushAttrib(GL_COLOR_BUFFER_BIT);
 		}
 
-		bind();
+		// Because we temporarily change global Fbo binding state
+		// when blitting from one framebuffer to the other, we first have 
+		// to query the renderer for its binding state, and store it.
+		GLuint rendererFboBindingState = ofGetGLRenderer()->getCurrentFramebufferId();
```

Comment:
Both read and write framebuffers are set in one instruction when you call `glBindFramebuffer` with the target `GL_FRAMEBUFFER`.

So, first, we have to bind read and write to different framebuffers in lines 943 and 944. Then, in line 949, **both** targets are restored to previous state =)

https://www.opengl.org/sdk/docs/man/html/glBindFramebuffer.xhtml